### PR TITLE
Feature/chart view

### PR DESCRIPTION
**Pull request by JeffShepherd**
### What Changed?
- Creation of chart view
- Creation of ChartView child components: IncidentInfo and PositionAssignment
- Installation of d3-org-chart library
### What's Next?
- Introduce logic that dynamically adds content for chart, as well as a dynamic search
### Known Issues?
- N/A
### Test Coverage?
- No tests added as components are likely to be updated with the next iteration

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,12 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (percworld) [inline public/index.html:17]: I haven't deleted this before and look forward to learning about why this metadata on mobile isn't preferred

File: src/components/App/App.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import Header from '../Header/Header';
 import Footer from '../Footer/Footer';
 import Profiles from '../Profiles/Profiles';
 import Passport from '../Passport/Passport';
+import ChartView from '../ChartView/ChartView';
```
Comment (percworld) [inline src/components/App/App.js:10]: Glad to see that this isn't throwing merge conflict errors anymore. This file's folder and imports were changed to uppercase and it appears to solve this problem.

File: src/components/ChartView/ChartView.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      'Planning Chief​': {attributes: {name:'Unassigned', title:'Planning Chief​', email:'', phone:''}}
+    }
+
+    //iterate through incident data and replace any default values with assignees
+    testObject.forEach(assignee => {
+      chartValues[assignee.attributes.title] = assignee;
+    })
+
+
+    return Object.values(chartValues).map((item) => {
```
Comment (percworld) [inline src/components/ChartView/ChartView.js:37]: Interested in a walkthrough on these object values.

**Review by percworld — COMMENTED** [review]
This seems like a solid approach to chartview and orgchart components.  Great work on this.  Looking forward to seeing it in our application and how it supports a fluid user flow.

File: src/components/App/App.js [hunk 4]
```diff
@@ -19,6 +20,8 @@ const App = () => {
           <Route path='/database' component={DatabaseMenu} />
           <Route path='/profiles' component={Profiles} />
           <Route path='/passport' component={Passport} />
+          {/* below is test-only for specific incident view */}
+          <Route path='/test' component={ChartView} />
         </Switch>
```
Comment (cbmackintosh) [inline src/components/App/App.js:25]: I'm guessing we'll need to pass an id into this component to be able to get the data from a network request?

File: src/components/ChartView/ChartView.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+              </div>`
+    };
+  })
+  }
+
+
+  useEffect(() => {
+    setData(createChartData())
+  }, []);
+
```
Comment (cbmackintosh) [inline src/components/ChartView/ChartView.js:98]: We'll probably end up using useEffect for the api call, and then have createChartData full from global state

File: src/components/PositionAssignment/PositionAssignment.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+      </form>
+      
+      <div className="search-results-container"></div>
+
+      <div className="assignee-search-result">
+        <p>Assignee name here</p>
+        <button>assign</button>
+      </div>
+
```
Comment (cbmackintosh) [inline src/components/PositionAssignment/PositionAssignment.js:35]: Nice. Thanks for building out these components per our discussions around redesign this afternoon

**Review by cbmackintosh — APPROVED** [review]
Looks great! Thanks for tackling this critical and extremely complex aspect of the project. I think I understand more or less how data is passed into the chart, but I might need you to walk me through the intricacies so we can strategize integrating with network requests.

**Pull request merged**